hazelcast-hibernate5/src/test/java/com/hazelcast/hibernate/CustomPropertiesTest.java

```diff
@@ -58,45 +48,6 @@
     @Rule
     public final ExpectedException exception = ExpectedException.none();
 
-    @Test
-    public void testNativeClient() throws Exception {
-        TestHazelcastFactory factory = new TestHazelcastFactory();
-        Config config = new ClasspathXmlConfig("hazelcast-custom.xml");
-        HazelcastInstance main = factory.newHazelcastInstance(config);
-        Properties props = getDefaultProperties();
-        props.remove(CacheEnvironment.CONFIG_FILE_PATH_LEGACY);
-        props.setProperty(Environment.CACHE_REGION_FACTORY, HazelcastCacheRegionFactory.class.getName());
-        props.setProperty(CacheEnvironment.USE_NATIVE_CLIENT, "true");
-        props.setProperty(CacheEnvironment.NATIVE_CLIENT_CLUSTER_NAME, "dev-custom");
-        props.setProperty(CacheEnvironment.CONFIG_FILE_PATH,"hazelcast-client-custom.xml");
-        HazelcastMockInstanceLoader loader = new HazelcastMockInstanceLoader();
-        loader.configure(props);
-        loader.setInstanceFactory(factory);
-        SessionFactory sf = createSessionFactory(props, loader);
-        final HazelcastInstance hz = HazelcastAccessor.getHazelcastInstance(sf);
-        assertTrue(hz instanceof HazelcastClientProxy);
-        assertEquals(1, main.getCluster().getMembers().size());
-        HazelcastClientProxy client = (HazelcastClientProxy) hz;
-        ClientConfig clientConfig = client.getClientConfig();
-        assertEquals("dev-custom", clientConfig.getClusterName());
-        assertTrue(clientConfig.getNetworkConfig().isSmartRouting());
-        assertTrue(clientConfig.getNetworkConfig().isRedoOperation());
-        factory.newHazelcastInstance(config);
-        assertEquals(2, hz.getCluster().getMembers().size());
-        main.shutdown();
```

Comment:
Cluster size assertions are removed in the new test since I believe increasing cluster size and verifying it should not be a part of client tests.